packs/alertlogic/actions/lib/get_scan_results.py

```diff
@@ -32,8 +32,10 @@ def GetScanResults(config, scan_exec_id, new_vulns=False, new_ports=False):
     Raises:
     ValueError: On lack of key in config.
     """
+    API_HOST = "api.alertlogic.net"
```

Comment:
I would define this in `actions.lib.base` or similar so it can be re-used in all the actions and doesn't need to be dupliated all over.

so in actions/lib/base.py:

```python
API_HOST = 'foo'
```

Then inside the actions:

```python
from lib.base import API_HOST
```